tests/validators/answers/test_number_answer_validator.py

```diff
@@ -9,8 +9,8 @@ def test_minimum_value():
         "id": "answer-4",
         "label": "Max/Min out of system limits",
         "mandatory": False,
-        "maximum": {"value": 99999999999},
-        "minimum": {"value": -99999999999},
+        "maximum": {"value": 99999999999999},
```

Comment:
Can you use the `_` separator here and below as well? It is hard to read.

Reply:
Added 👍